
From: "Martin J. Bligh" <mbligh@aracnet.com>

I don't think we really want experimental filesystems on by default.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/Kconfig.reiser4 |    1 -
 1 files changed, 1 deletion(-)

diff -puN fs/Kconfig.reiser4~stop-reiser4-from-turning-itself-on-by-default fs/Kconfig.reiser4
--- 25/fs/Kconfig.reiser4~stop-reiser4-from-turning-itself-on-by-default	Tue Sep  7 16:31:51 2004
+++ 25-akpm/fs/Kconfig.reiser4	Tue Sep  7 16:31:51 2004
@@ -1,7 +1,6 @@
 config REISER4_FS
 	tristate "Reiser4 (EXPERIMENTAL very fast general purpose filesystem)"
 	depends on EXPERIMENTAL && !4KSTACKS
-	default y
 	---help---
 	  Reiser4 is more than twice as fast for both reads and writes as
 	  ReiserFS V3, and is the fastest Linux filesystem, by a lot,
_
